compiler/plc_diagnostics/src/diagnostician.rs

```diff
@@ -135,17 +154,12 @@ impl Default for Diagnostician {
 pub trait DiagnosticAssessor {
     /// determines the severity of the given diagnostic
     fn assess(&self, d: &Diagnostic) -> Severity;
-}
-
-/// the default assessor will treat ImprovementSuggestions as warnings
-/// and everything else as errors
-#[derive(Default)]
-pub struct DefaultDiagnosticAssessor;
-
-impl DiagnosticAssessor for DefaultDiagnosticAssessor {
-    fn assess(&self, d: &Diagnostic) -> Severity {
-        //TODO: Refer to some severity map to reassign severity here.
-        d.get_severity()
+    //TODO should these be results
+    fn explain(&self, error: &str) -> String {
+        format!("Unknown error {error}")
+    }
+    fn get_diagnostic_configuration(&self) -> String {
+        "This assessor cannot be configured".into()
     }
```

Comment:
why implement these here?
its a bit strange that the assessor also explains.

Reply:
I can move this to the single impls as well